daemon/cni/cni.go

```diff
@@ -113,6 +118,11 @@ func Init() error {
 
 	conf["plugins"] = plugins
 
+	// TODO: check if we can avoid creating a custom file for propagating value of env INTER_NODE_LINK_TYPE
```

Comment:
I understand this is to teach CNI plugin to use gRPC instead of VXLAN, right? 
Would it make sense to add the cleanup logic for when daemonset is uninstalled/terminated?